src/templates/app/_includes/professional_client.jsx

```diff
@@ -4,10 +4,10 @@ import { Fieldset } from '../../_common/components/forms.jsx';
 const ProfessionalClient = () => (
     <Fieldset legend={it.L('Professional Client')} id='fs_professional' className='invisible'>
         <div className='gr-padding-10 gr-12'>
-            <p>{it.L('By default, all clients are treated as retail but anyone can request to be treated as professional.')}</p>
+            <p>{it.L('By default, all Binary.com clients are retail clients but anyone can request to be treated as a professional client.')}</p>
```

Comment:
we don't hardcode Binary.com, please use it.website_name as a placeholder instead